src/kernels/jupyter/commands/serverSelector.ts

```diff
@@ -64,6 +65,10 @@ export class JupyterServerSelectorCommand implements IDisposable {
     }
 
     private async clearJupyterUris(): Promise<void> {
+        // Clear out our saved passwords from this session as well
```

Comment:
@IanMatthewHuff not directly related to this PR: do we clear jupyter uri when removing a server from the server quick pick? It seems we should have a `clearJupyterUri(uri: string)` to clear the password connection info for a url and call that when users manually remove a url from the server list.

Reply:
@rebornix That's a good catch. I totally forgot that options. Follow up PR today.